### ISSUE TITLE
Add Middleware to the test cases

### ISSUE THREAD

**Issue by Dennis-Rosenbaum** [Prog-Party/dynamic-qr-backend#4]
Currently a lot of tests fail and are therefore skipped. Because we have added some behaviour to the middleware, but this middleware isn't added to the tests yet.

![image](https://github.com/user-attachments/assets/a73cd6d1-3429-472c-8d14-04a42b5036dc)

**Comment by jensdenbraber** [comment]
Good point: the scope of these tests is moved to the middleware. Lets move these tests to the middleware tests.
